### PR TITLE
Fixed incorrect version display of pg_bulkload

### PR DESCRIPTION
**Pull request by zwyan0**

### CODE REVIEW

File: lib/Makefile [hunk 1]
```diff
@@ -27,9 +27,16 @@ MODULE_big = pg_bulkload
 
 EXTENSION = pg_bulkload
 
+# set pg_bulkload version
+BULKLOAD_VERSION = 3.1.20
```
Comment (mikecaat) [inline lib/Makefile:31]: Can I get from here?
https://github.com/ossc-db/pg_bulkload/blob/2d9a22578ba57913d8f07960d2815cf93aa277a3/include/common.h#L15

File: SPECS/pg_bulkload-pg10.spec [hunk 2]
```diff
@@ -54,8 +54,8 @@ install -m 755 lib/pg_bulkload.so              %{buildroot}%{_libdir}/pg_bulkloa
 install -m 644 lib/pg_bulkload.sql             %{buildroot}%{pg_contribdir}/pg_bulkload.sql
 install -m 644 lib/uninstall_pg_bulkload.sql   %{buildroot}%{pg_contribdir}/uninstall_pg_bulkload.sql
 install -m 644 lib/pg_bulkload.control         %{buildroot}%{pg_extensiondir}/pg_bulkload.control
-install -m 644 lib/pg_bulkload--1.0.sql        %{buildroot}%{pg_extensiondir}/pg_bulkload--1.0.sql
-install -m 644 lib/pg_bulkload--unpackaged--1.0.sql         %{buildroot}%{pg_extensiondir}/pg_bulkload--unpackaged--1.0.sql
+install -m 644 lib/pg_bulkload--3.1.20.sql        %{buildroot}%{pg_extensiondir}/pg_bulkload--3.1.20.sql
```
Comment (mikecaat) [inline SPECS/pg_bulkload-pg10.spec:57]: Does is work to change `install -m 644 lib/pg_bulkload--${version}.sql` ?

File: lib/Makefile [hunk 3]
```diff
@@ -27,9 +27,16 @@ MODULE_big = pg_bulkload
...
+
 #supports both EXTENSION (for >=9.1) and without_EXTENSION (for <PG 9.1)
-DATA_built = pg_bulkload.sql
-DATA = pg_bulkload--1.0.sql pg_bulkload--unpackaged--1.0.sql uninstall_pg_bulkload.sql 
+DATA_built = pg_bulkload.sql \
```
Comment (mikecaat) [inline lib/Makefile:34]: Do we need pg_bulkload.sql and uninstall_pg_bulkload.sql?

File: SPECS/pg_bulkload-pg10.spec (hunk 2 above)
Comment (zwyan0) [reply SPECS/pg_bulkload-pg10.spec]: The file used to manually register the pg_bulkload schema and functions in postgresql. Someone might still be using it, so it is best not to delete it at this time.

File: lib/Makefile (hunk 1 above)
Comment (zwyan0) [reply lib/Makefile:31]: I will modify this code.

File: lib/Makefile (hunk 3 above)
Comment (zwyan0) [reply lib/Makefile:34]: The file used to manually register the pg_bulkload schema and functions in postgresql. Someone might still be using it, so it is best not to delete it at this time.

Comment (mikecaat) [reply lib/Makefile:34]: OK, understood. Thanks!

File: SPECS/pg_bulkload-pg10.spec (hunk 2 above)
Comment (zwyan0) [reply SPECS/pg_bulkload-pg10.spec]: I changed this code.

File: lib/Makefile (hunk 1 above)
Comment (zwyan0) [reply lib/Makefile]: I changed this code.

File: lib/Makefile [hunk 4]
```diff
@@ -71,3 +76,13 @@ subclean:
 	rm -f nbtsort.c
 
 LDFLAGS+=-Wl,--build-id
+
+pg_bulkload--$(BULKLOAD_VERSION).sql: pg_bulkload.sql.in
+	sed 's,MODULE_PATHNAME,$$libdir/$(MODULE_big),g' $< \
+	| sed 's,BULKLOAD_VERSION,$(BULKLOAD_VERSION),g' > $@
+
+pg_bulkload--unpackaged--$(BULKLOAD_VERSION).sql: pg_bulkload--unpackaged.sql.in
```
Comment (mikecaat) [inline lib/Makefile:84]: Do we really need it now? 

"unpackaged" seems to be used for extension updates, but now pg_bulkload doesn't support it.
https://www.postgresql.org/docs/12/extend-extensions.html

Comment (mikecaat) [reply lib/Makefile:84]: Additionally, is it better to add documentation how to upgrade pg_bulkload? Users need to drop extension and create extension.

File: .gitignore [hunk 5]
```diff
@@ -30,5 +30,6 @@ bin/results
 
 # Some more
 lib/exports.list
-lib/pg_bulkload.sql
+lib/*.sql
```
Comment (mikecaat) [inline .gitignore:33]: Is it better to change `lib/pg_bulkload--*.sql` ?

File: lib/Makefile (hunk 4 above)
Comment (zwyan0) [reply lib/Makefile]: Yes, you are right.
pg_bulkload don't support ALTER EXTENSION to update the version.
So, I removed the file, and added description how to upgrade pg_bulkload.

File: .gitignore (hunk 5 above)
Comment (zwyan0) [reply .gitignore]: Yes, It's better than previous. So I changed it.

File: SPECS/pg_bulkload-pg10.spec [hunk 6]
```diff
@@ -54,8 +54,8 @@ install -m 755 lib/pg_bulkload.so              %{buildroot}%{_libdir}/pg_bulkloa
 install -m 644 lib/pg_bulkload.sql             %{buildroot}%{pg_contribdir}/pg_bulkload.sql
 install -m 644 lib/uninstall_pg_bulkload.sql   %{buildroot}%{pg_contribdir}/uninstall_pg_bulkload.sql
 install -m 644 lib/pg_bulkload.control         %{buildroot}%{pg_extensiondir}/pg_bulkload.control
-install -m 644 lib/pg_bulkload--1.0.sql        %{buildroot}%{pg_extensiondir}/pg_bulkload--1.0.sql
-install -m 644 lib/pg_bulkload--unpackaged--1.0.sql         %{buildroot}%{pg_extensiondir}/pg_bulkload--unpackaged--1.0.sql
+install -m 644 lib/pg_bulkload--%{version}.sql        %{buildroot}%{pg_extensiondir}/pg_bulkload--%{version}.sql
+install -m 644 lib/pg_bulkload--unpackaged--%{version}.sql         %{buildroot}%{pg_extensiondir}/pg_bulkload--unpackaged--%{version}.sql
```
Comment (mikecaat) [inline SPECS/pg_bulkload-pg10.spec:58]: Do we need remove the lines too?

Comment (mikecaat) [reply SPECS/pg_bulkload-pg10.spec:58]: Please check if you can create the rpms using the specs.

Comment (zwyan0) [reply SPECS/pg_bulkload-pg10.spec]: No, we don't need these lines. So I removed its.
Then I tested create the rpms and it seems fine.

File: lib/pg_bulkload--unpackaged.sql.in [hunk 7]
```diff
@@ -1,4 +1,4 @@
-/* pg_bulkload/pg_bulkload--unpackaged--1.0.sql */
```
Comment (mikecaat) [inline lib/pg_bulkload--unpackaged.sql.in:1]: Don't we need the file now?

File: SPECS/pg_bulkload-pg10.spec (hunk 6 above)
Comment (mikecaat) [reply SPECS/pg_bulkload-pg10.spec]: Thanks!

File: lib/pg_bulkload--unpackaged.sql.in (hunk 7 above)
Comment (zwyan0) [reply lib/pg_bulkload--unpackaged.sql.in:1]: No, we don't need this file. I will delete it.

**Review by mikecaat — APPROVED** [review]
Thanks! It looks good to me.

**Pull request merged**